### PR TITLE
Milestone demo 1, 2, and 3 updates.

### PR DESCRIPTION
**Pull request by ApplebaumIan**
Please note that a lot of the order of the deliverables has changed.

### CODE REVIEW

**Review by harrisonfedor — APPROVED** [review]
lgtm!

**Review by mgclay — APPROVED** [review]
lgtm

File: documentation/docs/development-plan/schedule.md [hunk 1]
```diff
@@ -149,8 +149,6 @@ gantt
 
 - F4: User can view tasks
     - R1: User can view a list of their tasks from Canvas LMS.
```
Comment (mgclay) [inline documentation/docs/development-plan/schedule.md:151]: R1: User can view a list of their assignments from Canvas LMS

File: documentation/docs/development-plan/schedule.md [hunk 2]
```diff
@@ -128,108 +128,64 @@ gantt
...
+    - R4: User is able to view their account information.
+    - R5: Credentials are authenticated via JWT token.
+    - R6: First-time user will be sent a verification email.
 
-R: Credentials are authenticated via JWT token
+- F2: Pet Selection, Customization, and Care
+    - R1: User can choose between two virtual pets.
+    - R2: User can select between two different cats (color palette).
+    - R3: User can name their virtual pet.
+    - R4: Users can see a still image of their virtual pet at all times.
```
Comment (mgclay) [inline documentation/docs/development-plan/schedule.md:144]: R5: User sees the pet's mood change temporarily from neutral to happy when they click on it.

**Pull request merged**